src/nvapi_d3d.cpp

```diff
@@ -150,20 +153,47 @@ extern "C" {
         if (pGetSleepStatusParams->version != NV_GET_SLEEP_STATUS_PARAMS_VER1)
             return IncompatibleStructVersion(n);
 
-        if (!nvapiD3dInstance->IsReflexAvailable())
+        if (!nvapiD3dInstance->IsReflexAvailable(pDevice))
             return NoImplementation(n, alreadyLoggedNoLfx);
 
-        pGetSleepStatusParams->bLowLatencyMode = nvapiD3dInstance->IsReflexEnabled();
+        pGetSleepStatusParams->bLowLatencyMode = nvapiD3dInstance->IsLowLatencyEnabled();
+
         return Ok(n);
     }
 
     NvAPI_Status __cdecl NvAPI_D3D_GetLatency(IUnknown* pDev, NV_LATENCY_RESULT_PARAMS* pGetLatencyParams) {
+        constexpr auto n = __func__;
         static bool alreadyLogged = false;
-        return NoImplementation(__func__, alreadyLogged);
+
+        if (nvapiAdapterRegistry == nullptr)
+            return ApiNotInitialized(n);
+
+        if (pGetLatencyParams->version != NV_LATENCY_RESULT_PARAMS_VER1)
+            return IncompatibleStructVersion(n);
+
+        if (nvapiD3dInstance->IsUsingLfx() || !NvapiD3dLowLatencyDevice::SupportsLowLatency(pDev))
+            return NoImplementation(n, alreadyLogged);
+
+        NvapiD3dLowLatencyDevice::GetLatencyInfo(pDev, reinterpret_cast<D3D_LATENCY_RESULTS*>(pGetLatencyParams));
```

Comment:
I guess we should return `Error` instead  of `Ok` when `NvapiD3dLowLatencyDevice::GetLatencyInfo` returned false? The same applies to other reflex related`NvAPI_D3D_*` functions and also `NvAPI_D3D12_NotifyOutOfBandCommandQueue` and `NvAPI_D3D12_SetAsyncFrameMarker`.